composer.json

```diff
@@ -16,6 +16,7 @@
         }
     ],
     "require": {
+        "php": "~5.5|~7.0",
         "symfony/http-foundation": "~2.3|~3.0",
```

Comment:
this should be `2.3.*|^2.7.0|^3.0.0` i think. currently it allows 2.4 and 2.5 and 2.6 which is too permissive. same for the other symfony dependencies below.